automated-testing/cypress.env.json.prod-cannon

```diff
@@ -14,5 +14,6 @@
     {"token": "Stata", "name": "Stata"},
     {"token": "SAS", "name": "SAS"},
     {"token": "Matlab", "name": "Matlab"}
-  ]
+  ],
+  "support_ticket": {"creationEnabled": true, "queue": "Internal:Logging"}
```

Comment:
`support_ticket` default values can be set into https://github.com/hmdc/sid2/blob/canary/automated-testing/cypress/support/base.js#L33 for simplicity

Reply:
Yes, but since these are site-specific values, I prefer that they go in configuration files.